sql/variables/system_variables.go

```diff
@@ -1030,7 +1030,7 @@ var systemVars = map[string]sql.SystemVariable{
 		Dynamic:           false,
 		SetVarHintApplies: false,
 		// TODO: lower bound should be 0: https://github.com/dolthub/dolt/issues/7634
```

Comment:
Looks like this TODO has been completed and should be removed 

```suggestion
```

Reply:
Fixed.